services/src/docs/asciidoc/overview.adoc

```diff
@@ -8,5 +8,5 @@ Version: 1
 
 === URI scheme
 Host: localhost:8080
-BasePath: /auth
+BasePath: /auth/admin/realms
```

Comment:
Isn't this as follows under Quarkus?
```suggestion
BasePath: /admin/realms
```

Reply:
That's true with Quarkus, but squeezing out all `auth` paths in all of Keycloak would be a different thing I believe. I was  willing to approve it as is.

Reply:
Do we have an issue logged for this though? And if not, can you create one?

Reply:
Now we have an issue: #11256

Reply:
Nice, consider this resolved then!